sidecar/run-sidecar.sh

```diff
@@ -35,13 +35,14 @@ while true
   curl --max-time 10 -s -o out --request POST --data '{ "jwt": "'"${JWT}"'", "role": "'"${ROLE}"'"}' "${VAULT_ADDR}"/v1/auth/jwt/login
   TOKEN=$(cat out | jq -r  '.auth.client_token')
 
-  curl --max-time 10 -s -o config.json -H "X-Vault-Token: $TOKEN" $VAULT_ADDR/v1/secret/data/db-config/config.json
+  curl --max-time 10 -s -o temp -H "X-Vault-Token: $TOKEN" $VAULT_ADDR/v1/secret/data/db-config/config.json
   RT=$?
+  cat temp | jq -r ".data.data" > config.json
   if [ "$RT" == "0" ]; then
     mv config.json $CFGDIR/
   fi
 
-  curl -s -o temp -H "X-Vault-Token: $TOKEN" $VAULT_ADDR/v1/secret/data/db-config/config.ini
+  curl --max-time 10 -s -o temp -H "X-Vault-Token: $TOKEN" $VAULT_ADDR/v1/secret/data/db-config/config.ini
```

Comment:
Good idea to add the timeout. At this point, we can remove the 30 seconds of sleep. It was mostly done just for debugging.

Reply:
We should also remove the 'cat' calls since they would be exposing the secret information into the log. And please make a check whether the 'mv' was successful. For example, if the mount is read-only, it would fail, so we should log an error message. The same for `config.json`